src/box/lua/stat.c

```diff
@@ -67,6 +67,23 @@ inject_current_stat(struct lua_State *L, const char *name, size_t val)
 	lua_pop(L, 1);
 }
 
+/**
+ * Function creates table, which contains one field `total`
+ * with given @val. Puts this table to the field @name in
+ * the table which located on the top of the lua stack.
+ */
+static void
+inject_total_stat(struct lua_State *L, const char *name, size_t val)
```

Comment:
This function is named like inject_current_stat, but has a different effect. This is confusing.

Reply:
fixed